Makefile

```diff
@@ -51,11 +51,21 @@ endif
 # Rules for verification, formatting, linting, testing and cleaning #
 #####################################################################
 
+.PHONY: check
+check: $(GOIMPORTS) $(GOLANGCI_LINT) $(HELM) $(IMPORT_BOSS)
```

Comment:
There are no targets here for downloading these tools.
You can either drop the dependencies or add some targets to download the needed dependencies.

Reply:
ok, I added targets for the tools in use and removed the others